src/frontend/views/isbn-registry/forms/IsbnIsmnPublicationRegistrationForm.jsx

```diff
@@ -196,13 +196,13 @@ function IsbnIsmnPublicationRegistrationForm (props) {
       if(values.publicationFormat === FORMATS.PRINT_ELECTRONICAL) {
         return [...activeContent.fields, ...printFields, ...electronicalFields];
       }
-    // If publication type is dissertation and format is print or print and electronical, add additional fields regarding edition and manufacturer
+    // If publication type is dissertation and format is print or print and electronical, add additional fields regarding manufacturer
     } else {
-      const dissertationEditionField = activeContent.additionalFields.find(f => f.name === 'dissertationEdition').fields;
+      const dissertationAdditionalFields = activeContent.additionalFields.find(f => f.name === 'dissertationAdditionalFields').fields;
```

Comment:
Include optional chaining operator since not all active content has additionalFields property. Also since find may result into undefined you need to add conditional value of empty array to be able to use the spread operator as part of return value.